workflows/prognostic_c48_run/tests/test_radiation.py

```diff
@@ -153,13 +153,20 @@ def get_zarr(rundir, zarrname):
     return xr.open_zarr(zarrpath)
 
 
+# https://github.com/ai2cm/fv3net/pull/2388
+# We lost some combination of serialbox regression data stored
+# from a commit off of master on physics_standalone.  If we want
+# to recreate a regression test, we should redo this against
+# current outputs.
+@pytest.mark.skip(reason="missing data dependencies")
```

Comment:
Don't skip these tests -- they should pass now that we restored the forcing and lookup data to the bucket.

A follow-up idea here would be to move those forcing and lookup data to a bucket that doesn't sound like something archived from DSL since they're still in active use-- `vcm-ml-code-testing-data` presumably? I can copy this over and update the radiation package's hard-coded bucket path.

Reply:
Thanks.  Some of the data in `get_data.sh` was still missing, which is why I initially omitted.  Tests look like they pass now that I removed the `radiation_driver` data from the retrieval script.